coalib/output/ConsoleInteraction.py

```diff
@@ -304,25 +305,28 @@ def print_results_formatted(log_printer,
 
     for result in result_list:
         severity_str = RESULT_SEVERITY.__str__(result.severity)
+        args = vars(result)
         try:
             if len(result.affected_code) == 0:
+                args['affected_code'] = None
                 print(format_str.format(file=None,
                                         line=None,
                                         end_line=None,
                                         column=None,
                                         end_column=None,
                                         severity_str=severity_str,
-                                        **result.__dict__))
+                                        **args))
```

Comment:
The reason I made another variable `args` which contains the same `dict` from `result.__dict__` is because `result.affected_code` is `tuple` of `SourceRange` objects, and each `SourceRange` object in that tuple gets printed once, i.e in each iteration. **All the other values remain the same, just like in the previous implementation**. Otherwise, I have not changed anything.